Jafar_Alhajji/week_01/day_02/js_Homework/main.js

```diff
@@ -0,0 +1,217 @@
+//Homework 
+//What number's bigger?
+
+var numOne = 56;
+var numTwo = 77;
+console.log("Number 1 :" + numOne);
+console.log("Number 2 :" + numTwo);
+if (numOne > numTwo)
+{
+    console.log("Number " + numOne + " is greater than " +numTwo);
+}
+else if (numOne < numTwo)
+{
+    console.log("Number " + numTwo + " is grater than " +numOne);
+}
+else 
+{
+    console.log(numTwo + " = " +numOne);
+}
+
+//--------------
+//Driving Age
+
+var userAge = 16;
+var legalAge = 18;
+
+if (userAge >= legalAge)
+{
+    console.log("Drive away!");
+}
+else 
+{
+    console.log("Sorry, you can't drive yet");
+    console.log("Sorry, you have " + (legalAge - userAge) + " years to wait until you can drive")
+}
+
+//---------------
+
+//The World Translator
+
+var lang = "English";
+
+if (lang === "English")
+{
+    console.log("Hello World");
+}
+else if (lang === "Arabic")
+{
+    console.log("مرحبا بالعالم");
+}
+else 
+{
+    console.log("Hello World in your language");
+}
+
+//------------
+//Suburbs
+var suburb1 = "AL-hasa" ;
+var suburb2 = "Riyadh";
+
+if (suburb1 === "AL-hasa")
+{
+    console.log("because of the team I support");
+}
+else 
+{
+    console.log("because It's the capital");
+}
+
+//-------
+//The Grade Assigner
+var grade = 88;
+
+if (grade <= 100 && grade >=90)
+{
+    console.log("A");
+}
+else if (grade < 90 && grade >=80)
+{
+    console.log("B");
+}
+else if (grade < 80 && grade >=70)
+{
+    console.log("C");
+}
+else if (grade < 70 && grade >=60)
+{
+    console.log("D");
+}
+else if (grade < 60 && grade >=0)
+{
+    console.log("F");
+}
+else
+{
+    console.log("Wrong greade");
+}
+
+//--------------
+//Air Conditioning
+
+var currentTem = 35;
+var functional = true;
+var desiredtemp = 24;
+
+if (currentTem > desiredtemp &&  functional)
+{
+    console.log("Turn on the A/C Please.");
+}
+else if (currentTem > desiredtemp &&  !functional)
+{
+    console.log("Fix the A/C now! It's hot!");
+}
+else if (currentTem < desiredtemp &&  !functional)
+{
+    console.log("Fix the A/C whenever you have the chance... It's cool...");
+}
+else 
+{
+    console.log("notheing");
+}
+
+//-------------
+//Golf
+
+var par = 4;
+
+if (par === 1)
+{
+    console.log("Hole in one");
+}
+else if (par <= par - 2)
+{
+    console.log("Eagle");
+}
+else if (par === par - 2)
+{
+    console.log("Birdie");
+}
+else if (par === par)
+{
+    console.log("Par");
+}
+else if (par === par + 1)
+{
+    console.log("Bogey");
+}
+else if (par === par + 2)
+{
+    console.log("Double Bogey");
+}
+else if (par >= par + 3)
+{
+    console.log("Not sure");
+}
+else 
+{
+    console.log("notheing");
+}
+//---------------
+//Serge Says
+
+var userMessage = "?";
+
+if (userMessage.endsWith("?"))
+{
+    console.log("Sure");
+}
+else if (userMessage === userMessage.toLocaleUpperCase())
```

Comment:
`toLocaleUpperCase()` is a bit more complex method than `toUpperCase()`. It's fine to use it here, but it could cause problems when you start defining your own rules later on in the course. You can read more about the difference between `toUpperCase()` and `toLocaleUpperCase()` [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleUpperCase).